lib/chef-dk/policyfile/dsl.rb

```diff
@@ -118,6 +118,20 @@ def cookbook(name, *version_and_source_opts)
         end
       end
 
+      def metadata
+        cookbook_root = storage_config.relative_paths_root
+        unless File.exist?(File.join(cookbook_root, "metadata.rb")) || File.exist?(File.join(cookbook_root, "metadata.json"))
+          raise PolicyfileMissingCookbookMetadata
```

Comment:
Can this error include the paths we checked? In the basic case it's probably obvious, but once folks put 2 layers of scripts and such on top of it, it could get confusing.